news/9547.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed the bug preventing the use of --editable installs for setup.cfg-only setuptools-based projects.
```

Comment:
Suggestion: change this to a .feature.rest file, and change the content to: Add support for editable installs for project with only setup.cfg files.

That’ll make it more discoverable and most users (and I) would consider this a new feature.

Reply:
IMHO it is not quite a new feature, I was expecting that if `setuptools` understands `setup.cfg`-only projects, that `pip` would understand them too automatically, but it turned out that `pip` uses `setuptools` in a bit own way.

Reply:
I understand this is a bugfix from the technical perspective, but in practice, very few user *expected* `pip install -e` to work on a setup.py-less project before this change, and most would *feel* it is more like a new feature instead. The changelog is ultimately consumed mostly by people not faimilar with pip internals, so it’s probably better to mark this as a feature.